validator/client/beacon-api/duties_test.go

```diff
@@ -1029,17 +835,17 @@ func TestGetDuties_Valid(t *testing.T) {
 				nil,
 			).Times(2)
 
-			fetchSyncDuties := testCase.epoch >= params.BeaconConfig().AltairForkEpoch
-			if fetchSyncDuties {
-				dutiesProvider.EXPECT().SyncDuties(
-					ctx,
-					testCase.epoch,
-					validatorIndices,
-				).Return(
-					generateValidSyncDuties(pubkeys, validatorIndices),
-					nil,
-				).Times(2)
-			}
+			// fetchSyncDuties := testCase.epoch >= params.BeaconConfig().AltairForkEpoch
```

Comment:
Commented line left here.

Reply:
@sb-anderson 
This comment is not resolved yet